docs/content/en/docs/tasks/packages/prereq.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
+Before installing any curated packages for EKS Anywhere, do the following:
+
+* Check that the version of `eksctl anywhere` is `v0.9.0` or above with the following commands:
+    ```bash
+    eksctl anywhere version
+    ```
+    Example command output
+    ```
+    NAME                                       READY   STATUS     RESTARTS   AGE
+    eks-anywhere-packages-57778bc88f-587tq     2/2     Running    0          16h
+    ```
+* Make sure cert-manager is up and running in the cluster. Note cert-manager is not installed on workload clusters by default. If cert-manager is not installed, you can manually install cert-manager and follow the instructions below to finish the package controller installation.
+
+* Check the existence of package controller:
+
+    ```bash
+    kubectl get pods -n eksa-packages | grep "eks-anywhere-packages"
+    ```
+    If the returned result is empty, you need to install the package controller.
+
+* If the packages controller is not yet installed, follow these steps:
+
+    1. Install the package controller
+        ```bash
+        export CURATED_PACKAGES_SUPPORT=true
```

Comment:
feature flag CURATED_PACKAGES_SUPPORT is no longer needed after this release.

Reply:
@chrisnegus, I am working on doc changes which will be updating these items. I assume this PR is to just move items around and not update the new changes that will be associated with this release. @binyamz, the change specified above is not required now since I am working on those updates.

Reply:
@a-cool-train Yes. The intention is to just move items around. Would you prefer I close this PR and have you handle it or would you like to work with this as a base?

Reply:
We should make sure the docs are versioned properly so this step remains in docs that _do_ require this env

Reply:
@chrisnegus, Feel free to get this PR in, then I'll use it as a base.

Reply:
Okay. @a-cool-train I'll go ahead and merge this PR and let you make the fixes that are needed before the release.